workers/unity/Assets/Playground/Scripts/Bootstrap.cs

```diff
@@ -4,6 +4,9 @@
 using Improbable.Gdk.Core;
 using Unity.Entities;
 using UnityEngine;
+#if UNITY_ANDROID
+using Improbable.Gdk.Android;
```

Comment:
right now it doesn't look like we need this namespace anywhere in the bootstrap / or I'm just not seeing it

Reply:
DeviceInfo.IsEmulator() is in that namespace